Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -78,195 +81,234 @@ Copyright Notice
-   proposed to build a parallel, experimental, live IPv6 DNS root system
-   to discover the limits of DNS root name service and deliver useful
-   technical output.  Possible research agenda will be explored on this
-   testbed, covering several aspects (but not limited to):
+   By design, the Yeti testbed system serves the root zone published by
+   the IANA with only those structural modifications necessary to ensure
+   that it is able to function usefully in the Yeti testbed system
+   instead of the production Root Server system.  In particular, no
+   delegation for any top-level zone is changed, added or removed from
+   the IANA-published root zone to construct the root zone served by the
+   Yeti testbed system.  In this document, for clarity, we refer to the
+   zone derived from the IANA-published root zone as the Yeti-Root zone.
 
-   o  IPv6-only operation
+   This document presents a summary of the experiments carried out as
+   part of the Yeti DNS Project during its first two years of operation.
 
-   o  DNSSEC key rollover
+2.  Areas of Study
 
-   o  Renumbering issues
+   The experiments described in this document are grouped loosely into
+   multiple sets, each representing a somewhat distinct set of problem
+   statements.  These areas of study are summarised here in the form of
+   lists of questions to give context to the experiments that follow.
 
-   o  Scalability issues
+   The infrastructure on which the experiments have been run has also
+   involved significant effort to build, principally in its operational
+   aspects: the coordination of a large pool of volunteers in the
+   implementation of the testbed is itself an exercise with relevance to
+   the real world, and commentary on the corresponding logistics is also
+   included in this document.
 
-   o  Multiple zone file signers
+2.1.  Yeti-Root Zone Distribution
 
-   Starting from May 2015, three coordinators began to build this live
-   experimental environment and called for participants.  At the time of
-   writing, there are 25 Yeti root servers with 15 operators, and
-   experimental traffic from volunteers, universities, DNS vendors,
-   mirrored traffic, non-Yeti traffic, and RIPE Atlas probes.
+   o  What are the scaling properties of Yeti-Root zone distribution as
+      the number of Yeti-Root servers, Yeti-Root server instances or
+      intermediate distribution points increase?
 
-   Note that the Yeti DNS Project has complete fealty to IANA as the DNS
-   name space manager.  All IANA top-level domain names will be
-   precisely expressed in the Yeti DNS system, including all TLD data
-   and meta-data[Root-Zone-Database].  Therefore, the Yeti DNS Project
-   is never an "alternative root" in the usual sense of that term.  It
-   is expected to inform the IANA community by peer-reviewed science as
-   to future possibilities to consider for the IANA root DNS system.
+2.2.  Yeti-Root Server Names and Addressing
```

Comment:
So far, individual yeti server use seperate and independent name. It is flexible and easy to adopt DNSSEC Incrementally. We can borrow or refer some discussion from RSSAC document , you and me are both the co-authors in that document.

Regarding the renumber issue, current Yeti just recommand resolver use out-band approach to detect and fetch the NS list of root zone. I'm not sure how to make it sound. You can review the previous txt in 03 -version of this draft.